caravel/assets/javascripts/modules/caravel.js

```diff
@@ -48,6 +48,7 @@ var color = function () {
     var seen = {};
     return function (s) {
       if (!s) { return; }
```

Comment:
Maybe this should be ``if(s === undefined) { return; }`` so zero metrics are rendered?